README.md

```diff
@@ -7,366 +7,8 @@ The screenshot below shows how Bartholomew is implemented as the official Fermyo
-/home/my_bartholomew_installation/bartholomew/target/release/bart check blog/*
-```
-
-The output from the above command is as follows (albeit snipped a little to save space)
-
-```
-✅ content/blog/2022-02-08-hello-world.md
-✅ content/blog/bindle-what-is-it.md
-✅ content/blog/dockercon.md
-✅ content/blog/dont-repatriate-servers.md
-✅ content/blog/dotnet-wasi.md
-✅ content/blog/fermyon-discord.md
-✅ content/blog/finicky-whiskers-part-1-intro.md
-✅ content/blog/finicky-whiskers-part-2-fileserver.md
-✅ content/blog/finicky-whiskers-part-3-microservices.md
-✅ content/blog/finicky-whiskers-part-4-infrastructure.md
-
-/// snip ///
-
-✅ content/blog/spin-v03.md
-✅ content/blog/spin-webhooks.md
-✅ content/blog/tinygo-webassembly-favicon-server.md
-✅ content/blog/wasm-builders-announcement.md
-✅ content/blog/wasm-day.md
-✅ content/blog/wasm-wasi-wagi.md
-✅ content/blog/webassembly-component-model.md
-✅ content/blog/webassembly-languages.md
-✅ content/blog/webassembly-vs-containers.md
-✅ content/blog/why-and-how-wasm-cms-bartholomew.md
-```
-
-### The relationship between Bartholomew and Spin
-
-To run Bartholomew, you will need a Spin-capable runtime. For example, you can just download a recent release of [Spin](https://github.com/fermyon/spin) and put it on your `$PATH`.
-
-Then the `make serve` command can start it all up for you.
-
-### The relationship between Bartholomew and the Spin Fileserver
-
-Bartholomew uses an external file server called [Spin-Fileserver](https://github.com/fermyon/spin-fileserver).
-
-Please build the spin-fileserver and then copy the resulting `spin_static_fs.wasm` file into the `modules/` directory. This will ensure that you are running the latest release of the spin-fileserver.
-
-## Running Bartholomew
-
-With Spin:
-
-```
-$ spin up
-```
-
-With `make`:
-
-```
-$ make serve
-```
-
-For convenience, `make serve` builds the code, and then runs `spin up`.
-
-### Preview Mode
-
-By default, Bartholomew will not display content that is unpublished.
-Content is unpublished if either:
-
-- The article is marked `published = false` in its head
-- The article has a publish date in the future
-
-To view unpublished content, turn on `PREVIEW_MODE`.
-
-Spin:
-
-```
-$ spin up -e PREVIEW_MODE=1
-```
-
-Make:
-
-```
-$ PREVIEW_MODE=1 make serve
-```
-
-### The Page Cache
-
-Unless the environment variable `-e DISABLE_CACHE=1` is set, the first load of a site will create a cache
-of page metadata in `config/_cache.json`. This is an optimization to reduce the number of file IO operations
-Bartholomew needs to make.
-
-If you are actively developing content, we suggest setting `DISABLE_CACHE=1`. By default, the `Makefile`'s `make serve`
-target disables the cache, as `make serve` is assumed to be used only for developers.
```

Comment:
The documentation of the `PREVIEW_MODE` and `DISABLE_CACHE` does not seem to be a part of the restricted documentation. It would be nice to include these as well without completely erasing them.

Reply:
Hi @karthik2804 
Yes, these preview and cache sections have actually just been moved (not deleted altogether). The are now both part of the contributing section in the context of contributing to the docs and creating new content (which requires preview as part of the PR).
The main goal was to have just a pointer to the docs in the README. 
I am super excited about the possibility of getting linked headings implemented, because then we can point directly to a specific part of the docs like "page cache" from other docs and articles etc.